transport/src/main/java/io/netty/channel/ChannelHandlerAppender.java

```diff
@@ -17,12 +17,13 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.NoSuchElementException;
 
 /**
  * A {@link ChannelHandler} that appends the specified {@link ChannelHandler}s right next to itself.
- * By default, it removes itself from the {@link ChannelPipeline} once the specified {@link ChannelHandler}s
- * are added. Optionally, you can keep it in the {@link ChannelPipeline} by specifying a {@code boolean}
- * parameter at construction time.
+ * By default, it <strong>NOT</strong> removes itself from the {@link ChannelPipeline} once the specified
```

Comment:
`By default, it <strong>NOT</strong> removes` -> `By default, it does <strong>NOT</strong> remove`